components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,259 @@
++++
+title = "Chef Automate Integration App"
+draft = false
+
+gh_repo = "automate"
+
+[menu]
+  [menu.automate]
+    title = "Chef Automate Integration App"
+    parent = "automate/reference"
+    identifier = "automate/reference/servicenow_integration.md ServiceNow Integration"
+    weight = 50
++++
+
+[ServiceNow](https://www.servicenow.com/) provides cloud-based services that automate enterprise IT operations. ServiceNow specializes in IT service management (ITSM) applications and provides forms-based workflow application development. It supports third-party application and data integrations. The most common integrations are for configuration management database (CMDB), incident management, problem management, change management, user administration, and single sign-on authentication.
+
+The Chef Automate Integration App for ServiceNow enables the integration of Chef Automate in the ServiceNow. The Chef Automate integration is a ServiceNow-certified scoped application available from the ServiceNow store. It integrates existing Chef Automate infrastructure and compliance functionality with ServiceNow enterprise services. Once installed and configured, this integration enables Chef Automate to create and update a ServiceNow Configuration Management Database (CMDB) with data from nodes managed by Chef Automate. The Chef Automate Integration App enables Chef Automate to create and update client node records to the configuration management database (CMDB) tables of ServiceNow. Chef Automate aggregates information about infrastructure nodes, the Chef Infra Client runs, and Chef Compliance scans, helping you monitor your infrastructure in real time.
+
+{{< figure src="/images/automate/snow_integration_dataflow_diagram.png" alt="Data Flow Diagram">}}
```

Comment:
It would be nice to trim the two-ish pixel border from the top and left side of this image.